src/Flow/ETL/Transformer/RowTransformer.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Flow\ETL\Transformer;
+
+use Flow\ETL\Row;
+use Flow\ETL\Transformer;
+
+/**
+ * @psalm-immutable
+ */
+interface RowTransformer extends Transformer
+{
+    public function transformRow(Row $row) : Row;
```

Comment:
Same here, there might be a better name for this method, maybe `transformOne(Row $row): Row` ?